Readme.md

```diff
@@ -191,7 +191,8 @@ while (keys.length) {
 
 [crockfordconvention]: http://javascript.crockford.com/code.html
 
-## Use lowerCamelCase for variables, properties and function names
+## Naming Conventions
+### Use lowerCamelCase for variables, properties and function names
```

Comment:
Just to make this debate muddier.

From what I've read, most orgs use `camelCase` for functions and variables, and reserve `PascalCase` for Objects. [1][http://airbnb.io/javascript/#naming-conventions] [2][https://google.github.io/styleguide/javascriptguide.xml#Naming] [3][https://contribute.jquery.org/style-guide/js/#naming-conventions] [4][https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style#Methods] [5][http://javascript.crockford.com/code.html]  Node's convention breaks from this, and thus the debate.

My 2¢: 
* Local variables are usually too trivial to enforce a strict naming convention. Prefer descriptive names.
* Functions and Objects are used beyond the file and/or function scope, and at-a-glance check might help readability.
* I don't have a preference, as long as we aren't doing [szHungarianNotation][https://en.wikipedia.org/wiki/Hungarian_notation].